atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -171,22 +172,29 @@ def __init__(self, tasker, workflow, platforms, build_kwargs,
 
         self.worker_builds = []
 
-    def get_excluded_platforms(self):
-        df_dir = self.workflow.source.get_dockerfile_path()[1]
-        exclude_platforms = set()
-        exclude_path = os.path.join(df_dir, self.EXCLUDE_FILENAME)
-        if os.path.exists(exclude_path):
-            with open(exclude_path) as f:
-                for platform in f:
-                    platform = platform.strip()
-                    if not platform:
-                        continue
-                    exclude_platforms.add(platform)
-
-        return exclude_platforms
-
     def get_platforms(self):
-        return self.platforms - self.get_excluded_platforms()
+        df_dir = self.workflow.source.get_dockerfile_path()[1]
+        excluded_platforms = set()
+        container_path = os.path.join(df_dir, self.CONTAINER_FILENAME)
+        if os.path.exists(container_path):
+            with open(container_path) as f:
+                data = yaml.load(f)
+                if 'platforms' not in data:
+                    return self.platforms
+                for action, platforms in data['platforms'].items():
+                    if 'not' in action:
+                        if type(platforms) == str:
+                            excluded_platforms.add(platforms)
+                        else:
+                            for x in platforms:
+                                excluded_platforms.add(x)
+                    elif 'only' in action:
```

Comment:
I can see a use-case for having both `not` and `only`: one of them is temporary.

Should this be `if` instead of `elif`?